CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+#### Version 9.0.0
+* [BREAKING] Removed the in-list optimization functionality from the SDK.
+* [NEW] Added ability to dynamically switch Workspace configuration while in Admin mode. You can now switch to another spaces easily by simply navigating to the Preview panel in Admin mode and pointing the SDK configuration to another space.
```

Comment:
Grammar, please fix here and backport to source.

Change
```
You can now switch to another spaces
```

to
```
You can now switch to other spaces
```

Reply:
Updated.